src/scripts/components/NoticeMeta.js

```diff
@@ -1,15 +1,17 @@
-import * as moment from 'moment/moment';
+import { dateI18n } from '@wordpress/date';
 
 /**
  * The notice metadata, for example the source of the notification or the date
  *
  * @param {Object} props
- * @param {number} props.date   The date of the notification.
+ * @param {string} props.date   The date of the notification.
  * @param {string} props.source The source of the notification.
  */
 export const NoticeMeta = ( { date, source } ) => (
 	<p className="wp-notification-meta">
 		<span className="name">{ source }</span> { '\u2022 ' }
-		<span className="date">{ moment( date ).fromNow() }</span>
+		<span className="date">
+			{ dateI18n( 'l jS F Y - h:i A', date, true ) }
```

Comment:
Can you make a utility function that does this named `fromNow` or `timeFromNow`?

Reply:
Actually just converts the epoch to a readable time... Would it be better `toCurrentDate`?